testapps/SimpleApp/SimpleApp.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
```

Comment:
Hmm, is that something we typically test separately? I can see the value in having 1 single TFM test for tools because of how they're launched but I'm not seeing a reason to have a single targeting TFM for both netcoreapp2.0 and net461.